ts/store/reducers/wallet/payment.ts

```diff
@@ -93,7 +109,7 @@ const reducer = (
         // a verifica operation will generate a new codice contesto pagamento
         // effectively starting a new payment session, thus we also invalidate
         // the rest of the payment state
-        ...PAYMENT_INITIAL_STATE,
+        ...state,
```

Comment:
Are you sure of this change?

Reply:
@fpersico I believe that before this was done because the verification was the first operation performed, now before the verification the entry route is set, and in this way it was overwritten with the initial value from the verification. Maybe I can change it like this? 
<img width="550" alt="Schermata 2019-06-12 alle 13 29 45" src="https://user-images.githubusercontent.com/10615049/59347958-2e792900-8d16-11e9-88d9-870116397e82.png">

Reply:
@cloudify please check this

Reply:
@fpersico @cloudify improved in commit 17f6b33